eng/packaging.targets

```diff
@@ -288,4 +288,19 @@
     <Error Condition="'$(ServicingVersion)' == '0'" Text="ServicingVersion is set to 0 and it should be an increment of the patch version from the last released package." />
```

Comment:
This could be made better if we were able to see what we shipped last -- we have done this in the past (harvesting / package validation).  For example, if it could see that NuGet had a package with servicing version 3 we could make sure that this was set to 4, though this is less likely to be accidentally missed.

Reply:
It could but I recommend to not attempt doing that before we had a design discussion for it. Fetching additional data from nuget that can't be part of nuget's upfront restore network calls incurs risk.